torch/csrc/utils/python_arg_parser.cpp

```diff
@@ -70,7 +70,7 @@ FunctionParameter::FunctionParameter(const std::string& fmt, bool keyword_only)
 bool FunctionParameter::check(PyObject* obj) {
   switch (type_) {
     case ParameterType::TENSOR: {
-      return THPVariable_Check(obj);
+      return obj == Py_None || THPVariable_Check(obj);
```

Comment:
I don't think `None` checks should be part of this. They're handled for all types in the parsing.